### DISCUSSION TITLE
TinyMCE config

### DISCUSSION THREAD

**Discussion by SimonVillage** [directus/directus#15040] [Archive]
I am looking for a way to enable image captions: https://www.tiny.cloud/docs-4x/plugins/image/#image_caption

Can I overwrite the TineMCE config somehow?

Edit:
I tried overwriting the config in the admin panel but there is no caption field when enabling it.

Options overwrite:
```json
{
    "image_caption": true
}
```

**Comment by Pachat** [comment]
You may have a look at #5664

**Reply by SimonVillage** [reply]
Doesn't work. I tried adding
```json
{
    "image_caption": true
}
```

but it does not have any effect.

**Comment by licitdev** [comment]
@SimonHausdorf Image caption is currently unsupported, please refer to idea #14645.
